exercises/concept/freelancer-rates/freelancer-rates.js

```diff
@@ -1,8 +1,23 @@
 // @ts-check
 //
-// The line above enables type checking for this file. Various IDEs interpret
-// the @ts-check directive. It will give you helpful autocompletion when
-// implementing this exercise.
+// ☝🏽 The line above enables type checking for this file. Various IDEs interpret
+// the @ts-check directive. It will give you helpful autocompletion on the web
+// and supported IDEs when implementing this exercise. You don't need to
+// understand types, JSDoc, or TypeScript in order to complete this JavaScript
+// exercise, and can completely ignore this comment block and directive.
```

Comment:
Do you intend to add this comment to all concept exercises?

Reply:
The first one has a big block. All the. Things it unlocks this, then nothing.

Unless you have a different suggestion

Reply:
No that sounds fine!